pgtap/withPoints/withPointsDD/edge_cases/undirected_driving_side.pg

```diff
@@ -39,7 +34,7 @@ PREPARE q1 AS
 SELECT seq, node, edge, cost::TEXT, agg_cost::TEXT FROM pgr_withPointsDD(
     'SELECT id, source, target, cost, reverse_cost FROM edge_table ORDER BY id',
     'SELECT pid, edge_id, fraction, side from pointsOfInterest',
-    -1, 4.8, 'b', details := true, directed:=false)
+    -1, 4.8, driving_side := 'b', details := true, directed:=false)
```

Comment:
Note: if "driving side" is compulsory parameter, then it must not be a named parameter

Reply:
A: I am testing the old signatures.

Reasons:
- What `undirected_driving_side.pg` does is: 
verify that in an undirected graph, if the results of  `r`, `b`, `l` driving side are equal.
- I do the change as you mention [here](https://github.com/pgRouting/GSoC-pgRouting/issues/300#issuecomment-1638419933):
```
if graph is directed then valid values are 'R', 'L', 'r', 'l', 
else valid values 'B', 'b'
```
- Because of the change, now if graph is undirected, there is only one valid input `b`. So there won't be a similar test in 3.6.0. 
- What I did in this file is removing changes I made last week and keep it as the same to main repo.